client-templates/bitbucket-server/accept.json.sample

```diff
@@ -701,6 +701,17 @@
         "username": "${BITBUCKET_USERNAME}",
         "password": "${BITBUCKET_PASSWORD}"
       }
+    },
+    {
+      "//": "get a specific commit identified by the commit hash or name of a branch",
+      "method": "GET",
+      "path": "/rest/api/1.0/projects/:project/repos/:repo/commits/:sha",
```

Comment:
would prefer if this block was next to similar paths, it helps when reviewing

Reply:
You mean using `BITBUCKET_API` instead? For some reason didn't work

Reply:
no, I just meant moving it next to similar blocks `/rest/api/1.0/projects/:project/repos/:repo/...`

Personal preference, no need to update if it doesn't sound good 😄

Reply:
I agree with @carlos-snyk

Reply:
Got it. Will fix, no problem